File: 101companies/implementations/html5tree/server/classes/employee.php
<?php

    class Employee {
        
        public $id;
        public $name;
        public $address;
        public $salary;
        
        public function getId() {
            return $id;
        }
        
        public function getName() {
            return $name;
        }
        
        public function getAddress() {
            return $address;
        }
        
        public function getSalary() {
            return $salary;
        }
        
        public function setId($newId) {
            $this->id = $newId;
        }
        
        public function setName($name) {
            $this->name = $name;
        }
        
        public function setAddress($address) {
            $this->address = $address;
        }
        
        public function setSalary($salary) {
            $this->salary = $salary;
        }
    }
?>

File: oopm/eclipse/chrestomathy/oo/company/iterable/Employee.java
// (C) 2009 Ralf Laemmel

package oo.company.iterable;

/**
 * An employee has attributes for name and salary.
 */
public class Employee extends Unit {

	private String name;
	private double salary;
	
	public String getName() { return name; }
	public void setName(String name) { this.name = name; }
	public double getSalary() { return salary; }
	public void setSalary(double salary) { this.salary = salary; }

}
